src/functions.php

```diff
@@ -30,22 +31,13 @@ function fut_get_available_file_types() {
  */
 function fut_format_raw_custom_types( $file_data_raw ) {
 
-	$file_data   = [];
 	$description = isset( $file_data_raw['desc'] ) ? array_map( 'sanitize_text_field', $file_data_raw['desc'] ) : [];
 	$mime_types  = isset( $file_data_raw['mime'] ) ? array_map( 'sanitize_text_field', $file_data_raw['mime'] ) : [];
 	$extensions  = isset( $file_data_raw['ext'] ) ? array_map( 'sanitize_text_field', $file_data_raw['ext'] ) : [];
 
-	foreach ( $description as $key => $desc ) {
-		$file_data[ $key ]['desc'] = $desc;
-	}
-
-	foreach ( $mime_types as $key => $mime_type ) {
-		$file_data[ $key ]['mime'] = strpos( $mime_type, ',' ) === false ? $mime_type : array_filter( array_map( 'trim', explode( ',', $mime_type ) ) );
-	}
-
-	foreach ( $extensions as $key => $extension ) {
-		$file_data[ $key ]['ext'] = '.' . strtolower( ltrim( $extension, '.' ) );
-	}
+	$file_data = _update_file_data_description( [], $description );
```

Comment:
Let's use the same unique `fut` prefix for glocal functions, e.g. `_fut_update_file_data_description()`. 

Also, I prefer to add the `function_exists()` conditional check before declaration for preventing fatal errors if some plugin has the same functions.

Reply:
Also, I'm not sure that we need `$file_data` param for all new global functions.